superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterControls.tsx

```diff
@@ -128,7 +128,7 @@ const FilterControls: FC<FilterControlsProps> = ({
   const activeOverflowedFiltersInScope = useMemo(
     () =>
       overflowedFiltersInScope.filter(
-        filter => isNativeFilter(filter) && filter.dataMask.filterState?.value,
+        filter => isNativeFilter(filter) && filter.dataMask?.filterState?.value,
```

Comment:
Currently, there are `isNativeFilter` and `isFilterDivider` as the type guard, so I think we should introduce a new type guard for this use case, for example, `isFilterWithDataMask`.

Reply:
Fixed in https://github.com/apache/superset/pull/22307!